packages/auth/src/openapi/specs/id-provider.yaml

```diff
@@ -89,6 +91,8 @@ paths:
               description: Client finish endpoint
         '401':
           description: Unauthorized
+        '404':
+          description: Unknown interaction
```

Comment:
We can just have this as a generic "Not Found" for now. I made an issue to properly define error codes in this spec here:
- https://github.com/interledger/rafiki/issues/3029